components/atom/input/src/Input/Component/index.scss

```diff
@@ -60,6 +61,7 @@ $class-read-only: '#{$base-class}--readOnly';
   @each $type, $attr in $sizes-atom-input {
     &-#{$type} {
       height: $attr;
+      min-height: $attr;
```

Comment:
Strange... Shouldn't work only with the `height` attribute?

Reply:
Yes, it should, I'm keeping the `min-height` for compatibility reasons, just-in-case anyone relies on it.

However, as a general rule I don't think it's a good idea setting a fixed height for text/form elements. We should play only with line-height and padding, but inputs are a bit special with line-heights...